### ISSUE TITLE
[MAC] BlenderGIS 2.2.8 and Blender 4.02 not loading any of the maps

### ISSUE THREAD

**Issue by amitchaudhary82** [domlysz/BlenderGIS#868]
### Discussed in https://github.com/domlysz/BlenderGIS/discussions/867

<div type='discussions-op-text'>

<sup>Originally posted by **amitchaudhary82** March  2, 2024</sup>
Hello,
I have newly installed Blender 4.02 on my Macbook Pro (M2 version) and have installed Blender GIS pluging
Have also created the OpenTopography API key, tested with Swagger UI and connection is OK.

Now whenever I try in Blender the below steps
Create New Flle
Click on "GIS" => "Web Geodata" => "Basemap" => Google + Sattelite
now the software hangs, no layers are loaded, have to press "Esc" key to make the system come back

Have checked the Blender Logs file and it keeps saying the below

ERROR:BlenderGIS-master.core.basemaps.mapservice:885:Corrupted tile on cache
Traceback (most recent call last):
  File "/Users/amitchaudhary/Library/Application Support/Blender/4.0/scripts/addons/BlenderGIS-master/core/basemaps/mapservice.py", line 883, in getImage
    img = NpImage(data)
  File "/Users/amitchaudhary/Library/Application Support/Blender/4.0/scripts/addons/BlenderGIS-master/core/georaster/npimg.py", line 122, in __init__
    self.data = self._npFromBLOB(data)
  File "/Users/amitchaudhary/Library/Application Support/Blender/4.0/scripts/addons/BlenderGIS-master/core/georaster/npimg.py", line 242, in _npFromBLOB
    data = self._npFromImgIO(img)
  File "/Users/amitchaudhary/Library/Application Support/Blender/4.0/scripts/addons/BlenderGIS-master/core/georaster/npimg.py", line 258, in _npFromImgIO
    data = imageio.imread(img)
  File "/Users/amitchaudhary/Library/Application Support/Blender/4.0/scripts/addons/BlenderGIS-master/core/lib/imageio/core/functions.py", line 181, in imread
    reader = read(uri, format, 'i', **kwargs)
  File "/Users/amitchaudhary/Library/Application Support/Blender/4.0/scripts/addons/BlenderGIS-master/core/lib/imageio/core/functions.py", line 107, in get_reader
    raise ValueError('Could not find a format to read the specified file '
ValueError: Could not find a format to read the specified file in mode 'i'

Could someone please support here as I thought it should be "plug and play" kind of a thing

</div>

**Comment by amitchaudhary82** [comment]
as an additional info, I can see that .gpkg files are available in my Blender Cache folder and these files I can also open them in QGIS however when I just drag and drop them in Blender, they are not loaded
![GPKG file in Blender](https://github.com/domlysz/BlenderGIS/assets/38136186/013dea0b-f5ef-40a2-bb25-93d3319a507e)


![GPKG file in QGIS](https://github.com/domlysz/BlenderGIS/assets/38136186/da4e4b03-0bf3-4729-8a7e-faf98a15bbf7)


I have also tried to download Blender 3.6.9 and it has the same issue

**Comment by ngoosojk** [comment]
I am also experiencing the same issue with Ubuntu 22.04. Blender version is 4.0.2 and blenderGIS version is 2.2.8

**Comment by rmnblm** [comment]
This will solve your problem (at least it did for me): https://github.com/domlysz/BlenderGIS/issues/655#issuecomment-1197557235

**Comment by airbag00** [comment]
> This will solve your problem (at least it did for me): [#655 (comment)](https://github.com/domlysz/BlenderGIS/issues/655#issuecomment-1197557235)
> 
> ### TL;DR:
> Run:
> 
> ```
> brew install freeimage
> launchctl setenv IMAGEIO_FREEIMAGE_LIB /opt/homebrew/Cellar/freeimage/3.18.0/lib/libfreeimage.dylib
> ```
> 
> Then try again opening the basemap.

This worked for me!  blender 4.1 on a M1 mac mini running macos Sonoma.

**Comment by domlysz** [comment]
fixed https://github.com/domlysz/BlenderGIS/releases/tag/2210
